deploy/demo/quickstart/aad-pod-identity-quickstart.sh

```diff
@@ -0,0 +1,150 @@
+#!/bin/bash
+# created by @phillipgibson
+
+# function to validate JSON demo config file input
+validate_demo_config_file () {
+	echo "Validating the JSON demo config file input..."
+	if [ -f $1 ]; then # TODO - Check file exist and last 5 char is .json | && [ ${2: -5} == "*.json"]
+		#echo $1
+		echo "The JSON config file has been validated."
+		echo "Retrieving config file data..."
+		ASSETSRESOURCEGROUPNAME=$(jq '.ASSETSRESOURCEGROUPNAME' $1)
```

Comment:
You can use `jq -r` to get rid of the quotes so that you don't have to perform `${VARIABLE//\"/}` below.

Reply:
I attempted to try to do that and I ran into some issues. Let me re-test and see.